resources/crds/k8s-operator.alluxio.com_datasets.yaml

```diff
@@ -63,6 +63,8 @@ spec:
                 - credentials
                 - path
                 type: object
+              multipleMountPoints:
+                type: boolean
```

Comment:
so now we have two flavored deployment for fuse, one is named "dataset" the other one is named "multimounts" is it?

Reply:
What do you mean by `deployment` here?
The only difference is if mount.table.source is ETCD then the CLI doesn't have the ufs address. Other parts in the fuse template stays the same